src/ShaderGen.Tests/Resource.resx

```diff
@@ -0,0 +1,178 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this resource file is pretty overkill for what we need, and it's quite a bloated file (plus the backing file).

Reply:
It's auto-generated in VS, and is the way I've always handled big strings.  I'll submit an alternate design.

Reply:
Done